src/wazuh_modules/syscollector/syscollector_bsd.c

```diff
@@ -331,6 +331,10 @@ cJSON* sys_parse_pkg(const char * app_folder) {
                         char ** parts = OS_StrBreak('>', read_buff, 4);
                         char ** _parts = OS_StrBreak('<', parts[3], 2);
 
+                        if (strstr(_parts[0], " (") != NULL) {
+                            strtok(_parts[0], " (");
```

Comment:
This will break the string when found any of both delimiter characters (space and parentheses). You can replace this piece by something like:

```
char * delim = strstr(_parts[0], " (");
if (delim) {
    *delim = '\0';
}
```

To ensure the string is cut at the desired point.